src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,101 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private final int defaultSize;
+    private final double resizeFactor;
+    private int size;
+    private T[] elements;
+
+    public ArrayList() {
+        defaultSize = 10;
+        resizeFactor = 0.5;
```

Comment:
according to this comment it was supposed to do this changes:
![image](https://github.com/mate-academy/jv-array-list/assets/73041091/12cd5d58-c706-4022-b813-f6aab367134e)

```suggestion
    private static final int DEFAULT_SIZE = 10;
    private static final double RESIZE_FACTOR = 0.5;
    private int size;
    private T[] elements;

    public ArrayList() {
```